test/components/entity/data.spec.js

```diff
@@ -1,26 +1,52 @@
 import EntityData from '../../../src/components/entity/data.vue';
+import EntityUpdate from '../../../src/components/entity/update.vue';
 
 import useEntity from '../../../src/request-data/entity';
 
 import testData from '../../data';
+import { load } from '../../util/http';
 import { mergeMountOptions, mount } from '../../util/lifecycle';
+import { mockLogin } from '../../util/session';
 import { testRequestData } from '../../util/request-data';
 
 const mountComponent = (options = undefined) =>
   mount(EntityData, mergeMountOptions(options, {
     container: {
       requestData: testRequestData([useEntity], {
+        project: testData.extendedProjects.last(),
         dataset: testData.extendedDatasets.last(),
         entity: testData.extendedEntities.last()
       })
     }
   }));
 
 describe('EntityData', () => {
-  it('does not render if there are no properties', () => {
```

Comment:
Since the component is now always rendered, not conditionally, we don't need a test for this anymore.